staging/src/k8s.io/client-go/discovery/discovery_client_test.go

```diff
@@ -329,12 +329,16 @@ var returnedOpenAPI = openapi_v2.Document{
 func openapiSchemaFakeServer() (*httptest.Server, error) {
 	var sErr error
 	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-		if req.URL.Path != "/swagger-2.0.0.pb-v1" {
+		if req.URL.Path != "/openapi/v2" {
```

Comment:
Need to test that the fallback path works?

Reply:
Will add the tests for fallback path